backend/audit-trail-service/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+services:
+  audit-app:
+    container_name: audit_service_api
+    build:
+      context: ./
+      args:
+        - APP_ENV
+    command: npm run start
+    ports:
+      - 3003:3000
+    volumes:
+      - ./:/usr/src/app
+    env_file:  # Add this line to specify the environment file
+      - .env.local # Specify the path to your .env file
+
+  pgsqldb:
+    container_name: pgsqldb
+    image: postgres:16
+    env_file:
+      - .env.local
```

Comment:
It's standard to use separate and specific env file for database. Ex: database.env
Try not to mix backend env with database env

Reply:
Understood!